published paper: [2018 ISME -- Horizontal operon transfer, plasmids, and the evolution of photosynthesis in Rhodobacteraceae](https://www.nature.com/articles/s41396-018-0150-9)

# Methods

A phylogenomic species tree of 105 representative Rhodobacteraceae was calculated to document the distribution of 44 **photosynthetic** and 61 **non-photosynthetic** strains. *two part*

The major aim was the inference of a robust phylogenetic PGC tree by the concatenation of all conserved PGC genes.

**Comparisons of single-gene phylogenies with the PGC tree were performed to determine the extent of HGT**.

The comparison of the PGC tree with the species tree, which was calculated on the same set of 44 photosynthetic Rhodobacteraceae, provided the basis for a reliable differentiation between a strictly vertical evolution and HOT of the complete PGC.

The **treefixDTL software** was used to “fix” non-significant conflicts between both phylogenies.

Finally, the topological reconciliation of both trees with the program **NOTUNG** allowed a determination of the minimal number of authentic HOTs in the evolution of Rhodobacteracae.

## Phylogenetic analyses and tree reconciliation

For inferring organism trees, core-gene supermatrices of concatenated alignments of orthologous proteins were generated as previously described [Phylogenomics of Rhodobacteraceae reveals evolutionary adaptation to marine and non-marine habitats](https://www.nature.com/articles/ismej2016198).

Maximum likelihood (ML) trees were inferred from the supermatrices with [ExaML v3.0.19](https://academic.oup.com/bioinformatics/article/31/15/2577/188794) using maximum-parsimony starting trees, automated detection of the best substitution model, and 100 bootstrap replicates to estimate the statistical support of the internal nodes.

For collecting single genes, **BlastP** v2.7.1+ searches were performed at the NCBI and Integrated Microbial Genome & Microbiomes (IMG) web sites (https://www.ncbi.nlm.nih.gov/; https://img.jgi.doe.gov/).

Gene alignments were generated by [**Clustal Omega**](http://msb.embopress.org/content/7/1/539) and subsequently if necessary manually improved with the Edit option of the [**MUST package**](https://doi.org/10.1093/nar/21.22.5264).

Highly variable positions and positions with >50% gaps were automatically removed by the program [**G-blocks**](https://academic.oup.com/sysbio/article/56/4/564/1682121) implemented in the software package MUST.

The ML-based phylogenetic inference was conducted with [**RAxML v8.2.4**](https://academic.oup.com/bioinformatics/article/30/9/1312/238053) under a PROTGAMMALGF model; the rapid bootstrap option was used in conjunction with the [**autoMRE bootstopping criterion**](https://www.liebertpub.com/doi/10.1089/cmb.2009.0179) and subsequent search for the best tree.

[*How many bootstrap replicates are necessary?*](https://www.liebertpub.com/doi/10.1089/cmb.2009.0179)

Bayesian inference was done with the program [**PhyloBayes** v3.3 under a CAT-GTR+4Γ model](https://academic.oup.com/bioinformatics/article/25/17/2286/209366).

Two independent chains were run for 10,000 replicates and each tenth generation was sampled.

The burn-in was estimated using the program **“bpcomp”** of the PhyloBayes package; the two chains were considered as converged if the maximal difference between the bipartitions was **<0.25**.

[The significance of alternative topologies was estimated with approximately unbiased (AU) test](https://academic.oup.com/sysbio/article/51/3/492/1616895).

The [**treefixDTL software**](https://academic.oup.com/sysbio/article/62/1/110/1657799) version 1.0.2 was applied to fix the topology of each associate tree (i.e., operon tree or gene tree) under default settings except for an alpha value of 0.001 for the paired-site test and the model closest to PROTGAMMALGF available via treefixDTL (PROTGAMMAJTTF) as RAxML substitution model.

To reconstruct the types and numbers of the evolutionary events that explain the discrepancies, if any, between the final topologies, [NOTUNG version 2.9](https://academic.oup.com/bioinformatics/article/28/18/i409/246367) was run under default settings except for the permission of horizontal transfers and the use of a DTL (duplication-transfer-loss) cost matrix of 2-3-1, corresponding to default costs used by treefixDTL.

The freeware program FigTree v1.3.1 for MacIntosh OSX computers created by Andrew Rambaut (Institute of Evolutionary Biology, University of Edinburgh; http://tree.bio.ed.ac.uk/) was used to draw the circular version of the schematic tree in Figure 5.
